project/BuildSettings.scala

```diff
@@ -24,7 +24,8 @@ object BuildSettings {
       Wart.DefaultArguments // for constructors for PureConfig
     ),
     ThisBuild / scalafixDependencies ++= Seq(
-      Dependencies.scalazzi // https://github.com/scalaz/scalazzi
+      Dependencies.scalazzi, // https://github.com/scalaz/scalazzi
```

Comment:
The comments should either be next to the dependency definition or removed completely - I don't think they're needed.